pi5/pinctrl_test.go

```diff
@@ -44,121 +44,6 @@ func TestNewBoard(t *testing.T) {
-
-	val, err = parseCells(2, &byteArray)
-	test.That(t, err.Error(), test.ShouldContainSubstring, "but there aren't enough bytes to read in from inputted bytestream")
-	test.That(t, val, test.ShouldEqual, 0x0)
-
-	byteArray = []byte{0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08}
-	val, err = parseCells(2, &byteArray)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, val, test.ShouldEqual, 0x102030405060708)
-
-	// this parseCells() call should omit the first 32 bits from the final outputted value
-	byteArray = []byte{0xFF, 0xFF, 0xFF, 0xFF, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08}
-	val, err = parseCells(3, &byteArray)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, val, test.ShouldEqual, 0x102030405060708)
-}
-
-func TestGetAddrSizeCells(t *testing.T) {
-	// this should throw an error since there are no address/size cell files here
-	path := "./mock-device-tree/proc/device-tree/axi/pcie@120000/rp1/gpio@d0000"
-	addr, size, err := getNumAddrSizeCells(path)
-	test.That(t, err.Error(), test.ShouldContainSubstring, "trouble getting addr cells info for")
-	test.That(t, addr, test.ShouldEqual, 0)
-	test.That(t, size, test.ShouldEqual, 0)
-
-	path = "./mock-device-tree/proc/device-tree/axi/pcie@120000/rp1"
-	addr, size, err = getNumAddrSizeCells(path)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, addr, test.ShouldEqual, 2)
-	test.That(t, size, test.ShouldEqual, 2)
-
-	path = "./mock-device-tree/proc/device-tree/axi/pcie@120000"
-	addr, size, err = getNumAddrSizeCells(path)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, addr, test.ShouldEqual, 3)
-	test.That(t, size, test.ShouldEqual, 2)
-}
-
-func TestGetRegAddr(t *testing.T) {
-	path := "./mock-device-tree/proc/device-tree/axi/pcie@120000/rp1/gpio@d0000"
-	reg, err := getRegAddr(path, 2)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, reg, test.ShouldEqual, 0xC0400D0000)
-}
-
-func TestGetRangesAddr(t *testing.T) {
-	path := "./mock-device-tree/proc/device-tree/axi/pcie@120000/rp1"
-	ranges, err := getRangesAddr(path, 3, 2, 2)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, ranges[0].childAddr, test.ShouldEqual, 0x4000000002000000)
-	test.That(t, ranges[0].parentAddr, test.ShouldEqual, 0x0)
-	test.That(t, ranges[0].addrSpaceSize, test.ShouldEqual, 0x400000)
-
-	path = "./mock-device-tree/proc/device-tree/axi/pcie@120000"
-	ranges, err = getRangesAddr(path, 3, 2, 2)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, ranges[0].childAddr, test.ShouldEqual, 0x0)
-	test.That(t, ranges[0].parentAddr, test.ShouldEqual, 0x1F00000000)
-	test.That(t, ranges[0].addrSpaceSize, test.ShouldEqual, 0xFFFFFFFC)
-	test.That(t, ranges[1].childAddr, test.ShouldEqual, 0x400000000)
-	test.That(t, ranges[1].parentAddr, test.ShouldEqual, 0x1C00000000)
-	test.That(t, ranges[1].addrSpaceSize, test.ShouldEqual, 0x300000000)
-}
-
-func TestDeviceTreeParsing(t *testing.T) {
-	t.Run("test board setup", func(t *testing.T) {
-		TestEmptyBoard(t)
-		TestNewBoard(t)
-	})
```

Comment:
so this just ran all of our tests a second time when I was testing with `make test` so I think its safe to remove for this repo